spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe SessionsHelper, type: :helper do
+  let(:user) { create(:user) }
+
+  describe '#corrent_user' do
+    context 'ログイン中のユーザーがいない時' do
+      it 'nilを返す' do
+        expect(current_user.nil?).to be_truthy
```

Comment:
be_nil はいかがでしょう？
https://rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers#be_nil-instance_method

```suggestion
        expect(current_user).to be_nil
```

Reply:
https://github.com/cat2koban/kalendar/pull/40/commits/8d5e611d7468089ff46f1330f577fa617d2863c0 にて対応しました。